src/time.cr

```diff
@@ -275,76 +281,90 @@ struct Time
     end
   end
 
-  def self.now : self
+  # Returns the current time in the local time zone.
+  def self.now : Time
     new
   end
 
-  def self.utc_now : self
+  # Returns the current time in UTC time zone.
+  def self.utc_now : Time
     seconds, nanoseconds = compute_seconds_and_nanoseconds
     new(seconds: seconds, nanoseconds: nanoseconds, kind: Kind::Utc)
   end
 
-  def date
+  # Returns a copy of `self` with time-of-day components (hour, minute, ...) set to zero.
+  def date : Time
     Time.new(year, month, day, kind: kind)
   end
 
-  def year
+  # Returns the year number (in the Common Era).
+  def year : Int32
     year_month_day_day_year[0]
   end
 
-  def month
+  # Returns the month number of the year (`1..12`).
+  def month : Int32
     year_month_day_day_year[1]
   end
 
-  def day
+  # Returns the day number of the month (`1..31`).
+  def day : Int32
     year_month_day_day_year[2]
   end
 
-  def hour
+  # Returns the hour of the day (`0..23`).
+  def hour : Int32
     ((total_seconds % SECONDS_PER_DAY) / SECONDS_PER_HOUR).to_i
   end
 
-  def minute
+  # Returns the minute of the hour (`0..59`).
+  def minute : Int32
     ((total_seconds % SECONDS_PER_HOUR) / SECONDS_PER_MINUTE).to_i
   end
 
-  def second
+  # Returns the second of the minute (`0..59`).
```

Comment:
"(`0..59`, or `0..60` when there's a leap second)"? We mention leap years below so we should be consistent.

Reply:
Are you sure `60` can actually be returned? It doesn't look like, seeing that the method does a remainder.

Reply:
You're correct: https://stackoverflow.com/a/16539483/2035962.